RAW_RELEASE_NOTES.md

```diff
@@ -7,7 +7,7 @@ The idea is that this is a low friction way to add release notes along with code
 will make it substantially easier for the releaser to "linkify" all of the release notes in the
 final version.
 
-## 1.6.0
+## 1.6.0.dev
```

Comment:
Not sure this is necessary, the release notes are intended for the next tagged version, do no ned for the `.dev` suffix.

Reply:
I agree. This is my fault. Just drop `dev` suffix.